ODCRequest/js/RequestForm.js

```diff
@@ -229,6 +291,8 @@ define(["dojo/_base/declare",
               });
             },
           });
+
+          date_objects.push(datebox)
```

Comment:
bear in mind the use of `;` I have had a few headaches debugging the code for an error and this was the cause.

Reply:
I didn't realise that semi-colons actually meant anything in JS. But good to be consistent I agree, I'll add them